connector/spanmetricsconnector/config.go

```diff
@@ -31,6 +31,11 @@ type Dimension struct {
 	Default *string `mapstructure:"default"`
 }
 
+type InstrumentationScope struct {
+	Name    string  `mapstructure:"name"`
+	Default *string `mapstructure:"default"`
```

Comment:
Is a default really necessary?

Reply:
Well, I can remove and simplify this. Yes, I will do.